pkg/providers/aws/cluster_network_provider.go

```diff
@@ -26,10 +27,15 @@ import (
 	errorUtil "github.com/pkg/errors"
 )
 
+const (
+	DefaultRHMIVpcNameTagKey   = "Name"
+	DefaultRHMIVpcNameTagValue = "RHMI Cloud Resource VPC"
+)
+
 //todo we need to ensure we test that if `IsEnabled` is true, it remains true for consecutive reconciles
 
 type NetworkManager interface {
-	CreateNetwork() error
+	CreateNetwork(context.Context, client.Client, ec2iface.EC2API) (*ec2.Vpc, error)
 	DeleteNetwork() error
 	IsEnabled(context.Context, client.Client, ec2iface.EC2API) (bool, error)
```

Comment:
Looks like we're going to use the client and ec2 service in everything. Maybe we should move them in to the implementation struct?